web/src/settings_invites.ts

```diff
@@ -53,12 +59,22 @@ type Invite = z.output<typeof invite_schema> & {
     notify_referrer_on_join?: boolean;
 };
 
-const meta = {
+type Meta = {
+    loaded: boolean;
+    invites: Invite[];
+    invite_id?: number;
```

Comment:
I don't think we should store `invite_id` here. We can set `data-invite-id` attribute and use that to get the ID wherever needed.

Reply:
You can check whether the html element contains correct ID. It should work.